py3-shellingham.yaml

```diff
@@ -28,13 +33,31 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: cba059e7f29f731c2fdf9c8c3ea7921182b7d6c3
 
-  - name: Python Build
-    runs: |
-      python -m build
-      python3 -m installer -d "${{targets.destdir}}" dist/*.whl
-      find ${{targets.destdir}} -name "*.pyc" -exec rm -rf '{}' +
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/build-installer
```

Comment:
this is really wierd. you dropped py3-build and installer from the environment.contents.packages but then use build-installer ? how does that even work?

Reply:
Probably a mistake, I added `build-installer` pretty late in going through these deps when once of my packages didn't install correctly with the pip pipeline and started retroactively converting the old ones back.